server/graphql/v2/input/CommentCreateInput.js

```diff
@@ -18,5 +18,6 @@ export const CommentCreateInput = new GraphQLInputObjectType({
       deprecationReason: '2019-02-26: Please use the expense field',
     },
     ConversationId: { type: GraphQLString },
+    UpdateId: { type: GraphQLString },
```

Comment:
The pattern we used for `ConversationId` is not the right one. The latest best practice is to have a `*ReferenceInput` (ie. `ExpenseReferenceInput`) so that we can refer the entity either by it's old ID, new ID, or slug if it has one.

We should create an `UpdateReferenceInput`, and have this here instead:
```suggestion
    update: { type: UpdateReferenceInput },
```